tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -996,6 +996,25 @@ static RColumnInfoGroup GatherColumnInfos(const RDescriptorsComparison &descCmp,
    return res;
 }
 
+static void PrefillColumnMap(const ROOT::RNTupleDescriptor &desc, const ROOT::RFieldDescriptor &fieldDesc,
+                             ColumnIdMap_t &colIdMap, const std::string &prefix = "")
+{
+   std::string name = prefix + '.' + fieldDesc.GetFieldName();
```

Comment:
Is that the correct behavior if the prefix has the default value?  i.e. the name 'starting' with a period?

Reply:
Yes, it's the same behavior as [line 909](https://github.com/root-project/root/pull/18187/files#diff-f47e9d804f32081f46022555af6dd04fd77b8f6a389a175d953e693aeabc90bbL909). (since it's just an internal name it's fine if it looks a bit odd)

Reply:
Well, column names are not a thing in RNTuple, so in respect to what are they "invalid"?

Reply:
Oh yeah, that's right. I was thinking about field names, but in the end a field can have multiple columns etc. Fine from my side.

Reply:
> (since it's just an internal name it's fine if it looks a bit odd)

Are we guaranteed those names won't leak anywhere (eg. displaying meta-data information about the `RNTuple`)?

Reply:
They are printed if you run the Merger in ExtraVerbose mode, but that's mostly for our own debugging. In the future we may want to improve the ExtraVerbose mode messages if there is the need for that.